napari/layers/_markers_layer/model.py

```diff
@@ -51,7 +52,7 @@ class Markers(Layer):
     See vispy's marker visual docs for more details:
     http://api.vispy.org/en/latest/visuals.html#vispy.visuals.MarkersVisual
     """
-    def __init__(self, coords, symbol='o', size=10, edge_width=1,
+    def __init__(self, coords, symbol='DISC', size=10, edge_width=1,
```

Comment:
shouldn't this be lowercase? and what was wrong with having `o` as a valid option, I don't think we should change that

Reply:
The case doesn't really matter because I make them all upper case in the `Markers.symbol` setter, but it definitely looks better lower case so I will change it.

In terms of having `o` as a valid option, would you be okay with removing disc in favor of keeping `o` (`o` and `disc` map to the same symbol)?